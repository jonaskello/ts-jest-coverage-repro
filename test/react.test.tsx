import * as React from 'react';
import * as renderer from 'react-test-renderer';

test('Test react component', () => {
  const component = renderer.create(
    <div>Testing</div>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
