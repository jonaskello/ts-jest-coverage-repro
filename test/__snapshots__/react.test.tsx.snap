// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test react component 1`] = `
<div>
  Testing
</div>
`;
